src/components/index.js

```diff
@@ -2,11 +2,11 @@ export {default as WebexAvatar} from './WebexAvatar/WebexAvatar';
 export {default as WebexActivity} from './WebexActivity/WebexActivity';
 export {default as WebexActivityStream} from './WebexActivityStream/WebexActivityStream';
 export {default as WebexDataProvider, AdapterContext} from './WebexDataProvider/WebexDataProvider';
-export {default as WebexInMeeting} from './WebexInMeeting/WebexInMeeting';
-export {default as WebexInterstitialMeeting} from './WebexInterstitialMeeting/WebexInterstitialMeeting';
 export {default as WebexLocalMedia} from './WebexLocalMedia/WebexLocalMedia';
 export {default as WebexMeeting} from './WebexMeeting/WebexMeeting';
 export {default as WebexMeetingControl} from './WebexMeetingControl/WebexMeetingControl';
 export {default as WebexMeetingControls, MeetingContext} from './WebexMeetingControl/WebexMeetingControls';
 export {default as WebexMeetingInfo} from './WebexMeetingInfo/WebexMeetingInfo';
 export {default as WebexRemoteMedia} from './WebexRemoteMedia/WebexRemoteMedia';
+export {default as WebexInMeeting} from './WebexInMeeting/WebexInMeeting';
```

Comment:
Can you revert this change? We wanted to keep the exports alphabetica

Reply:
ok